client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
```

Comment:
Having empty lines between imports of the same type leaves a lot of noise. What do you think about removing these empty lines where possible? If in doubts, just have a look at some of the client code in Calypso.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/27836/commits/a093d10a6ce2ad3116c67022db1ac1d94300d5ce